tests/Feature/Api/V1/UserRecentlyPlayedGamesTest.php

```diff
@@ -40,35 +39,40 @@ public function testGetUserRecentlyPlayedGamesUnknownUser(): void
             ->assertJson([]);
     }
 
-    public function testGetUserRecentlyPlayedGames(): void
+    public function testGetUserRecentlyPlayedGamesX(): void
     {
-        /** @var System $system */
-        $system = System::factory()->create();
-        /** @var Game $game */
-        $game = Game::factory()->create([
-            'ConsoleID' => $system->ID,
+        $game = $this->seedGame(achievements: 3);
```

Comment:
`seedGame` is required for anything that regenerates metrics as the core achievement set also has to be populated.